docs/usage.md

```diff
@@ -3,130 +3,441 @@
+   - The assistant processes the escalation request and generates the incident accordingly.
+
+3. **Review Incident Creation**:
+   - Verify that the incident has been created and that all relevant details from the alert have been transferred. Ensure that the incident record accurately reflects the information from the alert.
+
+### Review Escalated Incidents
+
+- Confirm that the escalation process has been completed successfully and that the new incident record contains all necessary information. Review the incident to ensure it is properly documented and ready for further action.
+
+### Integrate Escalated Incidents into Workflow
+
+- Incorporate the escalated incident into your workflow for comprehensive management and response. Use the incident record to track progress, assign tasks, and coordinate responses.
+
+>[!Tip]
+>Regularly review and escalate high-priority alerts to ensure timely and effective response. Prompt escalation helps address critical issues and maintains security posture.
+
+Escalating alerts to incidents is a crucial process for managing significant security threats within FortiSOAR. By following the outlined steps and utilizing the FortiAI assistant’s capabilities, users can effectively promote alerts to incidents and manage them with appropriate detail and urgency.
+
+>[!Note]
+>Ensure that all necessary details from the alert are accurately included in the incident record to support effective investigation and resolution.
+
+## Setup War Room
+
+Setting up a war room within the FortiAI assistant involves creating a dedicated space for managing and responding to an incident. This functionality is crucial for coordinating efforts, sharing information, and making decisions during incident response.
+
+### Use Case Example: Establishing a War Room for a Data Breach Incident
+
+A common use case is setting up a war room for a significant incident, such as a data breach, where various teams need to collaborate and respond quickly. Creating a war room allows for streamlined communication and efficient management of the incident.
+
+### Setting Up a War Room Using FortiAI Assistant
+
+1. **Initiate War Room Setup**:
+   - Issue a command to set up a war room for the incident. For example:
+     > *Establish a war room for incident [INCIDENT-ID]*
+   - The assistant creates a new war room associated with the specified incident.
+
+2. **Provide Incident Details**:
+   - Specify the details of the incident for which the war room is to be set up. For example:
+     > *Create a war room for incident [INCIDENT-ID] involving [INCIDENT-DESCRIPTION]*
+   - The assistant processes the request and initiate the war room setup accordingly.
+
+3. **Review War Room Creation**:
+   - Verify that the war room has been successfully created and that all relevant details are included. Ensure that the war room is properly configured to support incident response activities.
```

Comment:
Add a note:
1. On prompt submit, the setup war room playbook will be triggered and will present manual input.
2. If any record is open while escalating an alert then manual input will be presented on that record else it will be notified at "Pending Tasks" besides "Settings" in FortiSOAR tool bar.

Reply:
Done